wsl.go

```diff
@@ -888,6 +920,21 @@ func (p *Processor) findRHS(node ast.Node) []string {
 	return rhs
 }
 
+func (p *Processor) isShortDecl(node ast.Node) bool {
+	if node == nil {
+		return false
+	}
+
+	switch t := node.(type) {
+	case *ast.AssignStmt:
+		if t.Tok == token.DEFINE {
+			return true
+		}
+	}
+
+	return false
```

Comment:
This is clear but not really needed yet so I think we should try to avoid pre-mature optimization. This could be written as 

```suggestion
	if t, ok := node.(*ast.AssignStmt); ok {
		return t.Tok == token.DEFINE
	}
	
	return false
```